src/inference/mh.jl

```diff
@@ -375,7 +375,14 @@ function propose!!(
 
     # Make a new transition.
     densitymodel = AMH.DensityModel(
-        Base.Fix1(LogDensityProblems.logdensity, Turing.LogDensityFunction(vi, model, DynamicPPL.SamplingContext(rng, spl)))
+        Base.Fix1(
```

Comment:
It seems MH sampler should be tested as well?

Reply:
Done! Though it seems to really struggle with here, even with the prior (despite sampling from the prior; am really confused about this..)